### ISSUE TITLE
json files generated with seed0 mismatch the ones you provide

### ISSUE THREAD

**Issue by Ze-Yang** [ucbdrive/few-shot-object-detection#25]
I notice some errors in the few-shot json files for coco. For instance, the json file `full_box_1shot_person_trainval.json` contains redundant images, expected 1 but 8. Moreover, I try to generate these files that are directly under folder `datasets/cocosplit/` with seed 0. However, it turns out different with those you provide. Hope for clarifying. Thanks.

**Comment by xinw1012** [comment]
Thanks for your interest. For the seed0 annotation for COCO, we didn't generate them ourselves. We used the preprocessed files provided by the [prior work](https://github.com/bingykang/Fewshot_Detection/tree/master/data/cocosplit) as seed0 for consistent comparison (we only converted the txt files into json).  Our script is used to provide splits of other random seeds used in our paper.  Hope it helps!
